ec2utils/ec2deprecateimg/ec2deprecateimg

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/python
+
+# Copyright 2015 SUSE LLC, Robert Schweikert
+#
+# This file is part of ec2deprecateimg.
+# ec2deprecateimg is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# 
+# ec2deprecateimg is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with ec2deprecateimg. If not, see <http://www.gnu.org/licenses/>.
+
+import boto
+import boto.ec2
+import argparse
+import os
+import sys
+
+#DEBUG -- Local test REMOVE
+sys.path.append('/work/rjschwei/devel/Enceladus/ec2utils/ec2deprecateimg/lib')
+
+import ec2utils.ec2deprecateimg as ec2depimg
+from ec2utils.ec2DepImgExceptions import *
+
+
+# Set up command line argument parsing
+argparse = argparse.ArgumentParser(description='Deprecate images in EC2')
+argparse.add_argument(
+    '-a', '--account',
+    dest='accountName',
+    help='Account to use',
+    metavar='ACCOUNT_NAME',
+    required=True
+)
+hlpMsg = 'The deprecation period, image will be tagged for removal '
+hlpMsg += 'on "now + deprecation perion", specified in months, default '
+hlpMsg += ' is 6 month (Optional)'
+argparse.add_argument(
+    '-d', '--deprecation-period',
+    default=6,
+    dest='depTime',
+    help=hlpMsg,
+    metavar='NUMBER_OF_MONTHS',
+    type=int
+)
+hlpMsg = 'Do not perform any action, print information about actions that '
+hlpMsg += 'would be performed instared (Optional)'
+argparse.add_argument(
+    '-n', '--dry-run',
+    action='store_true',
+    default=False,
+    dest='dryRun',
+    help=hlpMsg
+)
+argparse.add_argument(
+    '-f', '--file',
+    default = os.path.expanduser('~') + os.sep + '.ec2utils.conf',
+    dest='confFile',
+    help='Path to configuration file, default ~/.ec2utils.conf (Optional)',
+    metavar='CONFIG_FILE'
+)
+depImgIdentifier = argparse.add_mutually_exclusive_group(required=True)
```

Comment:
Where's the rest of the mutually exclusive group? I only see the one option here.